backend/test/acceptance/api.shoots.spec.js

```diff
@@ -66,17 +66,16 @@ module.exports = function ({ agent, sandbox, k8s, auth }) {
     const res = await agent
       .post(`/api/namespaces/${namespace}/shoots`)
       .set('cookie', await user.cookie)
-      .send({ metadata: {
-        name,
-        annotations: {
-          'garden.sapcloud.io/purpose': purpose
-        }
-      },
-      spec })
+      .send({
+        metadata: {
+          name,
+          namespace
+        },
+        spec
+      })
 
     expect(res).to.have.status(200)
     expect(res).to.be.json
-    expect(res.body.metadata).to.eql({ name, namespace, resourceVersion, annotations })
```

Comment:
why is this removed?

Reply:
you are right, probably too eager deleting code...